packages/commons-ui-core/package.json

```
@@ -0,0 +1,62 @@
+{
+  "name": "commons-ui-core",
+  "version": "2.0.0-alpha.0",
+  "author": "CodeForAfrica",
+  "license": "GPL-2.0-or-later",
+  "private": true,
+  "main": "src/index.js",
+  "scripts": {
+    "test": "jest",
+    "lint": "eslint --fix './'",
+    "clean": "rm -rf .turbo node_modules",
+    "storybook": "start-storybook",
+    "build-storybook": "build-storybook --quiet -o dist"
```

```suggestion
    "clean": "rm -rf .turbo node_modules dist",
    "storybook": "start-storybook",
    "build-storybook": "build-storybook --quiet -o dist"
```
Cleaning should delete `dist` folder as well I take it.